### PR TITLE
issue/131 - sol005 driver's compatibility change for lmctl

### LINKED ISSUE #131: sol003 compatibility with lmctl
Currently, lmctl supports ansible, kubernetes and sol003 drivers to create the resource package with the help of lmctl command.
Now, we have applied the consolidated changes which will allow to create a resource package for sol005 driver as well.
PR is already raised for the changes.
https://github.com/IBM/lmctl/pull/130

### PR DESCRIPTION
**Pull request by rkumar8j**
Fixes #131

### CODE REVIEW

**Comment by dvaccarosenna** [comment]
Please raise an issue on this repository for the change. `#3383` refers to an issue from an internal system which cannot be used as a reference here.

File: lmctl/project/handlers/brent2dot1/brent_src.py [hunk 1]
```diff
@@ -96,6 +96,20 @@ class Sol003LifecycleTree(files.Tree):
     def scripts_path(self):
         return self.resolve_relative_path(Sol003LifecycleTree.SCRIPTS_DIR_NAME)
 
+class Sol005LifecycleTree(files.Tree):
```
Comment (dvaccarosenna) [inline lmctl/project/handlers/brent2dot1/brent_src.py:99]: Sol005 doesn't need to be added to `brent2dot1` - this is a legacy format we no longer use. So you can remove all of the changes you made to files in `brent2dot1`.

**Review by dvaccarosenna — CHANGES_REQUESTED** [review]
Also need to ensure any documentation is updated. We at least need to add the new sol005 option to the command reference for `lmctl project create`: https://github.com/IBM/lmctl/blob/master/docs/command-reference/project/create.md#params

**Review by dvaccarosenna — CHANGES_REQUESTED** [review]
Code is ok now but the Travis CI check is failing. Looks like there are unit test failures which need to be resolved before this PR can be approved and merged.

**Comment by rkumar8j** [comment]
@dvaccarosenna : There is a minor change I have done in the unit test case file and now everything is on place. Please review the same from your end.

**Comment by dvaccarosenna** [comment]
> Also need to ensure any documentation is updated. We at least need to add the new sol005 option to the command reference for `lmctl project create`: https://github.com/IBM/lmctl/blob/master/docs/command-reference/project/create.md#params

Has this doc change been made?

**Comment by rkumar8j** [comment]
Hi @dvaccarosenna : One separate issue is already raised for document updates.
https://github.ibm.com/tnc/tnc-o-tracking/issues/3398

As of now, this PR can be approved and document will be updated with the above issue.

**Comment by dvaccarosenna** [comment]
Hi @rkumar8j let's not use internal issue links here, as this is a public repo. 
The issue you refer to is for updating the official product documentation but this PR needs to update the docs in this repo, located at https://github.com/IBM/lmctl/blob/master/docs. 

This forms the basis for the official documentation, but the changes need to be applied here first.

**Comment by rkumar8j** [comment]
Hi @dvaccarosenna : I have updated the document wrt sol005 driver. Kindly let me know if need any further update.

File: docs/command-reference/project/create.md [hunk 2]
```diff
@@ -31,10 +31,12 @@ lmctl project create [OPTIONS] [LOCATION]
 
 Params for the root Project take the format "param_name param_value", whilst params for Subprojects (those set on `--contains`) use the format "subproject_name.param_name param_value". 
 
-For example, the following sets the driver param to "ansible" for the root Project but "sol003" for the Subproject named A:
+For example, the following sets the driver param to "ansible" for the root Project but "sol003" & "sol005" for the Subproject named A:
```
Comment (dvaccarosenna) [inline docs/command-reference/project/create.md:34]: This update is not required as this is just an example (so can leave it as just sol003)

File: docs/command-reference/project/create.md [hunk 3]
```diff
@@ -31,10 +31,12 @@ lmctl project create [OPTIONS] [LOCATION]
...
 
 ```
 --contains Resource::brent A --param driver ansible --param A.driver sol003
+
+--contains Resource::brent A --param driver ansible --param A.driver sol005
```
Comment (dvaccarosenna) [inline docs/command-reference/project/create.md:39]: This update is not required as this is just an example (so can leave it as just sol003)

**Review by dvaccarosenna — CHANGES_REQUESTED** [review]
Just 2 updates to remove and then we're good

File: docs/command-reference/project/create.md (hunk 3 above)
Comment (rkumar8j) [reply docs/command-reference/project/create.md]: Hi @dvaccarosenna : I have done the changes. Please review.

**Review by dvaccarosenna — APPROVED** [review]

**Pull request merged**